src/components/Carousel.js

```diff
@@ -1,19 +1,100 @@
 import React from 'react';
-
+import PropTypes from 'prop-types';
 import './Carousel.scss';
 
-const Carousel = () => (
-  <div className="Carousel">
-    <ul className="Carousel__list">
-      <li><img src="./img/1.png" alt="1" /></li>
-      <li><img src="./img/1.png" alt="2" /></li>
-      <li><img src="./img/1.png" alt="3" /></li>
-      <li><img src="./img/1.png" alt="4" /></li>
-    </ul>
-
-    <button type="button">Prev</button>
-    <button type="button">Next</button>
-  </div>
-);
+class Carousel extends React.Component {
+  state = {
+    width: `${this.props.itemWidth * this.props.frameSize}px`,
+    translateX: 0,
+    leftButton: 'true',
```

Comment:
```suggestion
    leftButton: true,
```
Используй булевое значение